apps/website/src/styles/forms.css

```diff
@@ -71,6 +71,7 @@ button {
 
   .checkbox {
     inline-size: 20px;
+    min-width: 20px;
```

Comment:
was this causing issues on a certain browser? the `inline-size` _should_ be handling this, I would think

Reply:
In Chrome, at least, I found the checkbox for the sharing opt-out checkbox group was smaller than the TOC agreement checkbox group despite having the same markup. I think it came down to there being more text content in the former of the two checkbox groups. I tried things like `min-inline-size` and `flex-grow` but this worked best to keep it consistent.

Reply:
ah, gotcha. thanks!